stixcore/products/level0/quicklookL0.py

```diff
@@ -173,7 +174,8 @@ def from_levelb(cls, levelb, parent=''):
                    ssid=packets.ssid,
                    control=control,
                    data=data,
-                   idb_versions=idb_versions)
+                   idb_versions=idb_versions,
+                   packets=packets)
```

Comment:
So is this so you can directly pass in a list of packets rather than a `Product`?